### DISCUSSION TITLE
MDI application on Uno framework

### DISCUSSION THREAD

**Discussion by wgraaf** [unoplatform/uno#11124] [Q&A] [answered]
How would one go about creating a Muliple Document Interface application with UNO / WinUI 3 ?

**Comment by jeromelaban** [comment, marked as answer]
At this time, MDI is generally implemented using the [`TabView`](https://learn.microsoft.com/en-us/windows/apps/design/controls/tab-view). 

If you're looking for floating child windows like early Win32 days, you may need to implement this manually, using gestures, pointers and `Canvas` absolute placement.
